examples/data-sources/bloxone_dns_hosts/data-source.tf

```diff
@@ -0,0 +1,16 @@
+# Get all DNS Hosts
+data "bloxone_dns_hosts" "all_hosts" {}
+
+# Get DNS Host by name
+data "bloxone_dns_hosts" "dns_host_by_name" {
+  filters = {
+    name = "dns_host_by_name"
+  }
+}
+
+# Get DNS Hosts with the specific tags
+data "bloxone_dns_hosts" "all_dns_hosts_by_tags" {
```

Comment:
You are using a single tag in the example below
```suggestion
# Get DNS Hosts with the specific tags
data "bloxone_dns_hosts" "all_dns_hosts_by_tag" {
```

Reply:
done